cypress/e2e/transaction-import.cy.ts

```diff
@@ -55,6 +55,7 @@ describe('Transaction Import', () => {
     cy.get('li').contains('Create "Non Existing Account"')
     cy.getInputFor('Destination').clear().type('EDITED ACCOUNT{enter}')
     cy.getInputFor('Date').should('have.value', '2023-06-20')
+    cy.getInputFor('Available From').should('have.value', '2023-06')
```

Comment:
cypress fails here, which is what I would expect since **EDITED ACCOUNT** does not exist and the transaction therefore can't be recognized as income.

Am I overlooking something?